apps/epp-amend/translations/src/en/pages.json

```diff
@@ -5,17 +5,29 @@
   "application-submitted": {
     "confirmed": "Amendment submitted"
   },
+  "amend-contact-details": {
+    "header": "What are your contact details?"
+  },
   "confirm": {
     "header": "Check your answers",
     "subheader": "Review your answers below and amend if required before proceeding to declaration and payment.",
     "sections": {
       "amend-licence-number":{
         "header": "Licence details"
+      },
+      "amend-contact-details": {
```

Comment:
Hi Temi, I just noticed that if line 18th  and  15th  are left like in this way this is the output will be:
`pages.confirm.sections.Licence details.header`
![image](https://github.com/user-attachments/assets/6711ba8e-e9f5-4e11-a286-e2d0193eefbf)

Can you use the title "Licence details" for the first and "Contact details" for the second

Reply:
@PaolaDMadd-Pro i can't understand what you mean because i have a working code. What you trying to achieve. I don't have applicant date of birth in my code so where is the image coming from.

Reply:
I resolve this comment since I checked also on BranchURL  EPP-60 and is working fine.